dot/network/host.go

```diff
@@ -222,7 +217,7 @@ func (h *host) broadcast(msg Message) {
 }
 
 // broadcastExcluding sends a message to each connected peer except specified peer
-func (h *host) broadcastExcluding(msg Message, peer peer.ID) {
+func (h *host) broadcastExcluding(msg Message, peer peer.ID) { //nolint
```

Comment:
Would this work as an array? Excluding might be more than one peer in some cases

Reply:
yeah, we don't really use this right now, since we need to implement some sort of actual gossip mechanism, even if it's just random selection. it might be deleted in the future